setup/upgrade/considerations.md

```diff
@@ -32,3 +32,43 @@ upgrading, such as architecture updates and breaking changes.
 
 - v1.25 updates the username format to allow the use of alphanumeric character
   and hyphens. The length of the username can be 1-39 characters, inclusive.
+
+## Upgrading from versions prior to v1.21
+
+Users upgrading deployments that predate the release of v1.21 to v1.21 or later
+should update their Helm values file to reflect Coder's [updated schema]. More
+specifically, users must change the following values:
```

Comment:
Can we mention that `v1.21 -> v1.26` have backwards compatibility, while `v1.27` onward do not?

Reply:
@coadler does this sound ok?

`The Helm charts shipped with versions 1.21 through 1.26 are backward-compatible, while those shipping with v1.27 and later are not`

Reply:
sgtm!